x-pack/plugins/code/public/components/main/side_tabs.tsx

```diff
@@ -31,6 +32,13 @@ class CodeSideTabs extends React.PureComponent<RouteComponentProps<MainRoutePara
   }
 
   public get tabs() {
+    const { goto } = this.props.match.params;
+    let lineNumber;
+    if (goto) {
+      if (parseGoto(goto)) {
+        lineNumber = parseGoto(goto)!.line;
```

Comment:
why do we need the `!` when we have already checked its existence in the line above?

Reply:
type check would fail if `!` is missing